src/java.base/share/classes/java/time/format/Parsed.java

```diff
@@ -125,10 +126,21 @@
 final class Parsed implements TemporalAccessor {
     // some fields are accessed using package scope from DateTimeParseContext
 
+    final boolean onlyChronoField;
+    @SuppressWarnings("unchecked")
+    private Map<TemporalField, Long> createFieldValuesMap() {
+        if (onlyChronoField) {
+            return new HashMap<>();
+        } else {
```

Comment:
Is this condition inverted :thinking: seems like it but how do you get the performance numbers